### PR TITLE
doc: intellij/pycharm run configurations for OpenEdx with tutor

### PR DESCRIPTION
**Pull request by qasimgulzar**
Screenshot 2024-03-27 at 3 57 02 PM

[Debug Open edX with IntelliJ IDEA and Add a Python Remote SDK — Tutor documentation.pdf](https://github.com/overhangio/tutor/files/14772198/Debug.Open.edX.with.IntelliJ.IDEA.and.Add.a.Python.Remote.SDK.Tutor.documentation.pdf)

### CODE REVIEW

**Comment by qasimgulzar** [comment]
@regisb could you help reviewing this PR?

File: docs/debugger/intellij.rst [hunk 1]
```diff
@@ -0,0 +1,105 @@
+Debug Open edX with IntelliJ IDEA and Add a Python Remote SDK
```
Comment (DawoudSheraz) [inline docs/debugger/intellij.rst:1]: For clarity, this is specific to edx-platform, right? I think we should mention that explicitly in the heading.

File: docs/debugger/intellij.rst [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+==============================================================
+
+To debug Open edX with IntelliJ IDEA and add a Python remote SDK, follow these steps:
```
Comment (DawoudSheraz) [inline docs/debugger/intellij.rst:4]: We can specify in what cases we would need a Python Remote SDK. I expect it is needed for IntelliJ as a general but not for PyCharm?

File: docs/debugger/intellij.rst [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+
+Install Required Tools
+----------------------
+
+Make sure you have Docker and (IntelliJ IDEA or PyCharm) is installed on your system along with the Python and Docker plugin.
```
Comment (DawoudSheraz) [inline docs/debugger/intellij.rst:9]: nit: Docker and IntelliJ IDEA or PyCharm are installed

Comment (DawoudSheraz) [inline docs/debugger/intellij.rst:9]: Which Python and Docker plugins to be specific?

File: docs/debugger/intellij.rst [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+
+Setup OpenEdx with tutor
```
Comment (DawoudSheraz) [inline docs/debugger/intellij.rst:11]: nit: OpenEdx --> Open edX

File: docs/debugger/intellij.rst [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+------------------------
+
+Below are the few steps you need to follow inorder add remote sdk.
```
Comment (DawoudSheraz) [inline docs/debugger/intellij.rst:14]: nit: in order to add a remote sdk

**Comment by DawoudSheraz** [comment]
Closing this PR as this should not be in Tutor's official docs.

**Pull request closed without merging**